.github/workflows/env-checker-ci-schedule.yml

```diff
@@ -20,8 +20,8 @@ jobs:
       # macos-latest is 10.15. We need to test 11 as well.
       matrix:
         os: [windows-latest, macos-latest, macos-11, ubuntu-latest]
-        node-version: [14, 16, 18]
```

Comment:
Do we still support 14? If not, 14 can be removed?

Reply:
Removed node 14, thx!